live/app/routes/assessments/rating.js

```diff
@@ -3,11 +3,21 @@ import Route from '@ember/routing/route';
 export default Route.extend({
 
   afterModel(assessment) {
-    this.get('store').createRecord('assessment-result', { assessment }).save();
 
-    assessment.get('type') === 'CERTIFICATION' ?
-      this.transitionTo('certifications.results', assessment.get('certificationNumber'))
-      : this.transitionTo('assessments.results', assessment.get('id'));
-  }
+    return this.get('store')
+      .createRecord('assessment-result', { assessment })
+      .save()
+      .finally(() => {
+        switch (assessment.get('type')) {
+          case 'CERTIFICATION':
+            return this.transitionTo('certifications.results', assessment.get('certificationNumber'));
 
+          case 'SMART_PLACEMENT':
+            return this.transitionTo('campaigns.skill-review', assessment.get('id'));
+
+          default:
+            return this.transitionTo('assessments.results', assessment.get('id'));
```

Comment:
si on peut pas revenir dessus, utiliser `this.replaceWith()` pour les transitions